js/options.js

```diff
@@ -24,7 +24,16 @@ function save_options() {
 	var spamCount = document.getElementById('SpamCount').value;
 	var blockSpam = document.getElementById('BlockSpam').checked;
 	var omniSearch = document.getElementById('OmniSearch').checked;
+	//Quni Ticket Breakdown
 	var quniTickets = document.getElementById('QuniTickets').checked;
+	var siQueue = document.getElementById('SIQueue').checked;
+	var taQueue = document.getElementById('TAQueue').checked;
+	var three60Queue = document.getElementById('360Queue').checked;
+	var eeQueue = document.getElementById('EEQueue').checked;
+	var themesQueue = document.getElementById('ThemesQueue').checked;
+	var vocQueue = document.getElementById('VocQueue').checked;
+	var statQueue = document.getElementById('StatQueue').checked;
+	var intQueue = document.getElementById('IntQueue').checked;
```

Comment:
Yeah probably going to add "show" to the beginning of the variable names, i.e. showThemesQueue.
Indicates what it does a little better, and somewhat indicates that it's a boolean.